docs/databuilder-vs-cohortextractor.md

```diff
@@ -0,0 +1,53 @@
+# Data Builder and cohort-extractor
+## Missing features from cohort-extractor
+
+* Many features of cohort-extractor are not yet implemented in Data
+  Builder.
+    * The current development approach is to implement a few features in Data
+      Builder fully end-to-end.
+    * See the [ehrQL reference](ehrql-reference.md) for a complete list
+      of supported features.
+* Data Builder has no current way to generate dummy data.
```

Comment:
suggestion: As mentioned elsewhere in this review, it does now, and we may want a page to link to describe it.